pkgs/development/python-modules/zeep/default.nix

```diff
@@ -44,21 +45,20 @@ buildPythonPackage rec {
     six
   ];
 
-  # testtools dependency not supported for py3k
-  doCheck = !isPy3k;
-
   checkInputs = [
     tornado
   ];
 
-  buildInputs = if isPy3k then [] else [
+  buildInputs = [
+    aiohttp
+    aioresponses
     freezegun
     mock
     nose
     pretend
-    pytest
-    pytestcov
     requests-mock
+    pytest_3
+    pytestcov
   ];
 
   patchPhase = ''
```

Comment:
Can you make this `postPatch` while you're here?
If we were to ever add a `patches` to this they'd never get applied otherwise.

Reply:
OK